src/tools/index.js

```diff
@@ -129,6 +129,13 @@ export default {
         link: 'http://requestmap.webperf.tools/',
         title: 'Request Map Generator',
         type: [external]
+      },
+      {
+        description:
+          'Linting tool that will help your site with accessibility, speed, security and more, by checking your code for best practices and common errors',
```

Comment:
One tiny nit, slightly smaller screen sizes show the text being kinda squished

<img width="326" alt="screen shot 2018-08-30 at 5 03 37 pm" src="https://user-images.githubusercontent.com/12476932/44885617-b35af080-ac76-11e8-8ed9-f750142b3c1d.png">

We definitely need to find a better solution to make cards larger 😅, but for the meantime can you shorten this a bit?

For example: "Linting tool that can help improve accessibility, speed and more by checking for best practices"